js/scripts/zone_loaders/mountain_zone_loaders.js

```diff
@@ -0,0 +1,154 @@
+const loadMountainSlope1 = () => {
+    const queueAssets = () => {
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.clouds.CLOUD_JUST_CLOUD_DARK.SPRITESHEET);
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.clouds.CLOUD_BUSHY_DARK.SPRITESHEET);
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.clouds.CLOUD_LANKY_DARK.SPRITESHEET);
+
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.flowers.TALL_PURPLE_FLOWER_1.SPRITESHEET);
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.grass.GRASS_1.SPRITESHEET);
+
+        ASSET_MGR.queueDownload(Decoration.DECORATIONS.trees.SPRUCE_1.SPRITESHEET);
+
+        ASSET_MGR.queueDownload(Precipitation.SPRITESHEET);
+    };
+
+    const addEntities = () => {
+
+        GAME.addEntity(new Border(
+            new Vector(ZONE.MIN_PT.x, 0),
+            new Vector(1, ZONE.PIXEL_SIZE.y),
+            Zone.getZones().village.woods
+        ));
+        GAME.addEntity(new Border(
+            new Vector(0, 0),
+            new Vector(ZONE.PIXEL_SIZE.x, 1),
+            Zone.getZones().mountain.slope2
+        ));
+    
+        TilemapInterpreter.setTilemap(mountainSlope1TileMap, false);
+        WeatherSystem.setWeather("snow", 1, "day");
+        
+        const groundLevel = 92;
+
+        // trees
+        GAME.addEntity(new Decoration(Decoration.DECORATIONS.trees.SPRUCE_5, Vector.blockToWorldSpace(new Vector(-15, groundLevel))), -1);
```

Comment:
If you can find a pattern it might help to put some of these in a for loop.

Reply:
Great suggestion! Unfortunately it's completely inconsistent so a for loop isn't super feasible here.